GameDesign/Assets/Scripts/MonoBehaviours/CustomerMonoBehavior.cs

```diff
@@ -71,4 +96,52 @@ bool hasMask()
     {
         return true;
     }
+
+    private class Unmasked: State{
+        private CustomerMonoBehavior _cmb;
+        public Unmasked(string name, CustomerMonoBehavior cmb) : base(name){
+            _cmb = cmb;
+        }
+
+        public override void Enter(){
+            Debug.Log("Enter unmasked");
+        }
+
+        public override void Tik(){}
+
+        public override void Exit(){}
+
+    }
+
```

Comment:
maybe moving the states to different files could be a good idea.

Reply:
why

Reply:
just for organization